Project.toml

```
@@ -56,7 +56,7 @@ RecipesBase = "1.1"
 Reexport = "1.0"
 Requires = "1.1"
 Setfield = "0.8"
-StartUpDG = "0.12.3"
+StartUpDG = "~0.12.3"
```

```suggestion
StartUpDG = "0.12.3"
```
Why? From the Pkg docs: "For all versions with a major version of 0 the tilde and caret specifiers are equivalent."

---

Oops, I didn't see that section. Thanks!